authors/apps/comments/views.py

```diff
@@ -126,3 +127,29 @@ def validate_user(self, comment_user, user):
             raise exceptions.ValidationError()
         else:
             return True
+
+
+class CommentHistoryView(views.APIView):
+
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, slug, pk):
+        try:
+            find_article(slug)
```

Comment:
Nice work on the implementation of the feedback to find if an article exists by slug.